gnovm/tests/files/assign_unnamed_type/more_filetest/recover6_filetest.gno

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"errors"
+)
+
+func main() {
+	println(f(false))
+	println(f(true))
+}
+
+func f(dopanic bool) (err error) {
+	defer func() {
+		if x := recover(); x != nil {
+
+			err = x.(error)
+
+		}
+	}()
+	q(dopanic)
+	return
+}
+
+func q(dopanic bool) {
+	if dopanic {
+		panic(errors.New("wtf"))
+	}
+}
+
+// Output:
+// undefined
```

Comment:
I think both the type and value would evaluate to `nil` if this was a go program. Is it expected to print `undefined` when evaluated by gno?

Reply:
Yes, that is expected. If TypedValue == nil or TypeValue.T == nil. println() It prints undefined.

println(a) in gno is like a customized fmt.Printf("%#v",a)  not as a pure println() in go.